stdlib/ballerina-database/src/main/ballerina/sql/Package.md

```diff
@@ -0,0 +1,21 @@
+## Package overview
+
+This package provides the common record types and constants required for other data management packages such as jdbc, mysql, h2. 
+
+### PoolOptions 
+
+The PoolOptions type the properties which are used to configure DB connection pool. This is used with jdbc, mysql and h2 endpoints to configure the connection pool associated with the endpoint.
+
+### SQLType
+
+The SQLType represents the SQL data type of a given parameter. When using a parameter the correct sql type which matches with the actual database table column type should be used. Otherwise data loss can be occured.
```

Comment:
Change "When using a parameter the correct sql type which matches with the actual database table column type should be used." to "When using a parameter, use the same SQL type as the actual database table column type."
Change "Otherwise data loss can be occured." to "Otherwise, data loss can occur."